_includes/v19.2/sql/movr-start.md

```diff
@@ -0,0 +1,6 @@
+- Run [`cockroach demo movr`](cockroach-demo.html) to open an interactive SQL shell to a temporary, in-memory cluster with the `movr` database preloaded and set as the [current database](sql-name-resolution.html#current-database).
+- Use [`cockroach workload`](cockroach-workload.html):
+ 1. Start up a [secure](secure-a-cluster.html) or [insecure](start-a-local-cluster.html) local cluster.
+ 1. Run `cockroach workload init movr` with the appropriate flags and [connection string](connection-parameters.html) to initialize and populate the `movr` database on your running cluster.
+ 1. Open an interactive SQL shell to the cluster with the [`cockroach sql`](use-the-built-in-sql-client.html) command.
```

Comment:
Same comment as in some other places re: could we show the exact `$ cockroach sql --insecure ...` command?  Would make it easier for users to copypasta and get through the instructions

Reply:
Sure thing! Question: Should we be providing copy-paste commands for users trying to start a secure cluster, or just for insecure clusters? We would have to make the assumption that folks have already created certs, etc, or we would have to provide sample commands for all of those steps...

Reply:
As we discussed, my vote is for simple insecure-only, since if you're using secure we can't really know what your connection string is.